behave/features/11.e2e_nhs_login.feature

```diff
@@ -0,0 +1,55 @@
+@core
+Feature: COVID-19 Shielded vulnerable people service - partial user journey - postcode not eligible (not valid postcode)
```

Comment:
does the feature title need a reference to nhs login?

Reply:
Oh, good spot I'll update